src/styles/components/quota/styles.less

```diff
@@ -26,3 +30,13 @@
     }
   }
 }
+
+.quota-info {
```

Comment:
Please don't override ui-kit styles. Designers understand that we'll always use ui-kit styles, even when their mockups are inconsistent with ui-kit